docs/index.rst

```diff
@@ -12,14 +12,19 @@ What is astroplan?
 **astroplan** is an open source Python package that helps astronomers plan
 observations.
 
-We're just getting started, so for now check out `Astropy`_, `PyEphem`_ or
+We're still in development, so for now check out `Astropy`_, `PyEphem`_ or
```

Comment:
You could instead make this a warning to the user that the API could change because this is still very much in active development (re-phrase it in your own words). You can make a warning by doing:

```
.. warning::

    Text for the warning
```

and sphinx will render it nicely.